server/src/analyzer.ts

```diff
@@ -0,0 +1,27 @@
+import { Dolos, FileView } from "@dodona/dolos/dist/dolos";
```

Comment:
I've tried to import directly, but I ran into a lot of issues with yarn workspaces when building the feature: tsc can't resolve the module.

Reply:
Weird. It is not that important for this prototype because we won't be distributing this, but in the future we might want to check whether this also works when packaged.